test/values.test.js

```diff
@@ -150,6 +150,49 @@ describe('Values', () => {
     expect(json.parseJSON(test_query_json)).toEqual(test_query)
   })
 
+  test('parse version query agnostic of object order', () => {
+    var simpleQuery = new Query({
+      lambda: 'x',
+      expr: { var: 'x' },
+      api_version: '3',
+    })
+
+    var complexQuery = new Query({
+      lambda: '_',
+      expr: { match: { index: 'idx' }, terms: ['str', 10] },
+      api_version: '3',
+    })
+
+    // simpleQuery assertions
+    var x = '{"@query":{"api_version":"3","lambda":"x","expr":{"var":"x"}}}'
+    expect(json.parseJSON(x)).toEqual(simpleQuery)
+    expect(json.parseJSON(x).toString()).toEqual(simpleQuery.toString())
```

Comment:
@marrony Would it be better to have these `.toString()` assertion in the `pretty print` test block below?

Reply:
I think the `.toString()` assertions here are not necessary, if you parsed the json and it equals to what you expect, then the `.toString()` would be the same

Reply:
@marrony That's what I was thinking as well -- just trying to cover each case as well as I can. I'll remove these 👍